kieserver/modules/kieserver/module.yaml

```diff
@@ -29,16 +29,16 @@ envs:
 artifacts:
```

Comment:
jbpm-wb-kie-server-backend-7.52.0.Final-redhat-00004.jar has to be replaced by
jbpm-wb-kie-server-backend-7.52.0.Final-redhat-00007.jar